R/initModel.R

```diff
@@ -36,28 +36,40 @@ initModel <- function(config = NULL,
     dir.create(outputFolder)
   }
 
-  cfg <- readConfig(config)
-  title <- cfg[["title"]]
-
-  if (is.null(path)) {
-    stamp <- format(Sys.time(), "_%Y-%m-%d_%H.%M.%S")
-    path <- file.path(outputFolder, paste0(title, stamp))
+  # Generate SLURM configuration if sending to SLURM
+  if (sendToSlurm) {
+    slurmConfig <- setSlurmConfig(slurmQOS = slurmQOS, tasks32 = tasks32)
   }
 
-  # Check if the given path already exists
-  if (file.exists(path)) {
+  # Check if an already existing path was given
+  if (!is.null(path) && file.exists(path)) {
```

Comment:
Are you sure, `path` can be `NULL` right now? I would expect this line to fail as `file.exists` cannot handle it. So I guess we need an outer if clause with `!is.null(path)`

Reply:
This works. Sorry for bothering you.